### PR TITLE
added a dockerfile and update depedencies

### PR DESCRIPTION
**Pull request by jochen42**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            ttf-freefont \
+            fontconfig \
+            dbus \
+    && \
+
+    # Install wkhtmltopdf from `testing` repository
+    apk add qt5-qtbase-dev \
+            wkhtmltopdf \
+            --no-cache \
+            --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ \
```
Comment (pepakriz) [inline Dockerfile]: why not https variant?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    chmod +x /usr/bin/wkhtmltopdf
+
+
+COPY . /app
+WORKDIR /app
+
+ENV PORT=80
+
+# install sys deps
+RUN apk add --update bash
```
Comment (pepakriz) [inline Dockerfile]: Maybe this can be installed in first RUN section to reduce docker layers.

BTW: is bash really necessary?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    mv /usr/bin/wkhtmltopdf /usr/bin/wkhtmltopdf-origin && \
+    echo $'#!/usr/bin/env sh\n\
+Xvfb :0 -screen 0 1024x768x24 -ac +extension GLX +render -noreset & \n\
+DISPLAY=:0.0 wkhtmltopdf-origin $@ \n\
+killall Xvfb\
+' > /usr/bin/wkhtmltopdf && \
+    chmod +x /usr/bin/wkhtmltopdf
+
+
+COPY . /app
```
Comment (pepakriz) [inline Dockerfile]: Please copy files just before `npm install`. (https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#build-cache)

Comment (pepakriz) [reply Dockerfile]: and EXPOSE instruction move up for better build process

**Review by pepakriz — CHANGES_REQUESTED** [review]
Good job! Thank you very much. I have some questions and ideas, but I'm going to merge it :+1:

File: Dockerfile (hunk 2 above)
Comment (jochen42) [reply Dockerfile]: yes. node-wkhtmltopdf spawn's it. see https://github.com/devongovett/node-wkhtmltopdf/blob/master/index.js#L201

i will move it to the first RUN

File: Dockerfile (hunk 3 above)
Comment (jochen42) [reply Dockerfile]: make sense. i will update

File: Dockerfile (hunk 1 above)
Comment (jochen42) [reply Dockerfile]: without reason :) but make's sense. i will update

**Comment by jochen42** [comment]
ok. just have a look now...

**Comment by pepakriz** [comment]
I fixed some typos and formatting issues and now it's in master :+1: 

Thanks!

**Pull request closed without merging**